### PR TITLE
1836 18scan updates

### PR DESCRIPTION
**Pull request by timbovelander**

### CODE REVIEW

File: src/data/games/1836.json [hunk 1]
```diff
@@ -1,19 +1,19 @@
 {
   "info": {
     "title": "1836",
-    "designer": "John Bohrer",
+    "designer": "Eddie Robbins",
     "background": "gray",
     "orientation": "horizontal",
     "titleX": 500,
-    "currency": "USD"
+    "currency": "$#"
```
Comment (kelsin) [inline src/data/games/1836.json]: Why was this needed? Normally USD works (at least for me), was it not for you?

Comment (timbovelander) [reply src/data/games/1836.json]: It did not look good on the player table when I put it on the map (Firefox):
afbeelding
